### ISSUE TITLE
wownero subaccounts 4.20

### ISSUE THREAD

**Issue by nahuhh** [cake-tech/cake_wallet#1764]
**Describe the bug**
wownero shows an "account" button on ghe receive screen (causes crash), but wow doesnt support subaccounts afaik

there is also accounts on the main screen that allows you to tap the wallet name to create a new sub account. This _does_ (unexpectedly) work

**Comment by MrCyjaneK** [comment]
this is fixed and subaccounts work in wow

**Comment by nahuhh** [comment]
> this is fixed and subaccounts work in wow

They should technically work, but wow devs had said in past that wow doesn't use them (perhaps just a design choice)

**Comment by MrCyjaneK** [comment]
yep, but the feature is there in API :p so you can use cake to store secret wownerinos that wowlet won't know about
